components/utils/useSessionGroups.ts

```diff
@@ -63,56 +63,48 @@ export function useSessionGroups(sessions: Session[]): SessionGroups {
     [Conference.Date.toString(), sessions.length],
   )
 
-  const { IsInProgress } = getConferenceDates(Conference, dateTimeProvider.now())
```

Comment:
It looks like without this the schedule is showing up outside of the conference date. We probably only what this component working when it's on the day